src/config/inflation.json

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
why do we need to duplicate this instead of just loading an ini file copied from the catapult-client?

Reply:
I was missing out on the `receipt` endpoint and was able to return a list of inflation.
`inflation.json` will be removed.

> just loading an ini file copied from the catapult-client
how does it work?
where can we find **ini** file?

Reply:
https://github.com/symbol/symbol-node-configurator/blob/main/templates/resources/config-inflation.properties